app/controllers/users_controller.rb

```diff
@@ -1,7 +1,7 @@
 class UsersController < Devise::RegistrationsController
   # https://github.com/CanCanCommunity/cancancan/wiki/authorizing-controller-actions
   helper_method :error_detector
-  load_and_authorize_resource :user, only: [:index, :show, :edit, :destroy, :confirm_destroy, :dashboard]
```

Comment:
Show is no longer authorized?

Reply:
yep. Other people want to see your profile